Package.swift

```diff
@@ -104,6 +104,9 @@ let package = Package(
         .target(
             name: "RollbarAUL",
             dependencies: ["RollbarCommon", "RollbarNotifier"],
+            platforms: [
+                .macOS(.v10_15),
```

Comment:
Nit: any reason to exclude Big Sur?

Reply:
that means "starting from" version and excludes everything older than 10.15. So, the Big Sur is honored :)